### ISSUE TITLE
WCAG Farbkontrast der Alerts mit style="info"

### ISSUE THREAD

**Issue by lukasniebler** [RRZE-Webteam/rrze-elements#244]
Anscheinend ist der Farbkontrast der alten Alerts mit Style="info" nicht hoch genug um die nachgeschärften Anforderungen für AAA zu erfüllen.

z.B. Hier sichtbar: https://my2.siteimprove.com/Inspector/1136762/A11Y/Page?pageId=99997411272&impmd=3DF0A3E50EC9CF0E4EE290ED53EF8D10#/sia-r66/failed/VW0yUkE4b0M=;YzAwZjgxYjQ=
